README.md

```diff
@@ -37,6 +37,7 @@ to! Here are some tips:
 - Returning users of LALRPOP may benefit from the [cheat sheet].
 - The [advanced setup] chapter shows how to configure other aspects of LALRPOP's
   preprocessing.
+- docs.rs API documentation for [lalrpop](https://docs.rs/lalrpop/latest/lalrpop/) and [lalrpop-util]
```

Comment:
Is there a corresponding link for lalrpop-util as well?

Reply:
Oh never mind, it is linked all the way at the end

Reply:
Yeah, to clarify the confusion on that, I tried linking them both at the bottom, there's already a "LALRPOP" link and I guess markdown links are case insensitive, so it was linking to the lalrpop grammar instead.

I suppose I could put the lalrpop-util link in line as well for consistency